Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/publish.py

```diff
@@ -52,9 +52,14 @@ def parser(cls, parser, loggers=None):
             '--user', dest='user', type=str, default=None,
             help="Run 'git push' as the specified user with https. Program will prompt for a password.",
         )
+        parser.add_argument(
+            '--exclude', action='append', type=str, default=[],
+            help='Exclude inferred branch or tag from publication',
+        )
 
     @classmethod
-    def branches_on(cls, repository, ref):
+    def branches_on(cls, repository, ref, exclude=None):
+        exclude = exclude or []
```

Comment:
With `--exclude` having a default value in the parser, is this default-value logic actually needed?

Reply:
It is not, will change.